bin/rbenv-update

```diff
@@ -1,23 +1,24 @@
 #!/usr/bin/env bash
-# Usage: rbenv update [--noop]
+# Usage: rbenv update [--noop|--quiet|--silent]
```

Comment:
I think I'd rather see only --quiet in the usage help. `--silent` can be an undocumented alias? (Having them both listed in usage implies that they do different things)

Also, need --verbose added here.